auth/src/main/java/org/aerogear/android/ags/auth/credentials/OIDCCredentials.java

```diff
@@ -0,0 +1,206 @@
+package org.aerogear.android.ags.auth.credentials;
+
+import android.util.Log;
+
+import net.openid.appauth.AuthState;
+
+import org.aerogear.android.ags.auth.AuthenticationException;
+import org.jose4j.jwa.AlgorithmConstraints;
+import org.jose4j.jws.AlgorithmIdentifiers;
+import org.jose4j.jwt.JwtClaims;
+import org.jose4j.jwt.consumer.InvalidJwtException;
+import org.jose4j.jwt.consumer.JwtConsumer;
+import org.jose4j.jwt.consumer.JwtConsumerBuilder;
+import org.jose4j.keys.RsaKeyUtil;
+import org.jose4j.lang.JoseException;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+
+/**
+ * Credentials for OIDC based authentication
+ */
+public class OIDCCredentials implements ICredential {
```

Comment:
Add `@Override` where appropriate.

Reply:
`ICredential` is just empty at the moment.